lib/utils.js

```diff
@@ -1,6 +1,19 @@
 // Custom Splitter:
-const split = (str, delimiter) => {
-    return (str.split(delimiter || ',').map(element => element.trim())).filter(element => element);
+const split = (str, delimiter, whitespace) => {
+
+    if (whitespace === undefined && typeof delimiter === 'boolean') {
```

Comment:
I dislike this - parameters should represent what they actually are. If it's being called wrong, we should fix the callers, not address wrong calls in this way.

Reply:
Updated!